ui/src/components/TaskDetailsCard/__snapshots__/index.test.jsx.snap

```diff
@@ -32,7 +32,7 @@ exports[`should render TaskDetailsCard 1`] = `
                 class="MuiTypography-root MuiListItemText-secondary MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-displayBlock"
               >
                 <button
-                  class="MuiButtonBase-root MuiButton-root MuiButton-text Label-mini-17 MuiButton-textSizeSmall MuiButton-sizeSmall Label-dense-16 Mui-disabled Label-info-23 Label-disabled-18 Mui-disabled"
+                  class="MuiButtonBase-root MuiButton-root MuiButton-text Label-mini-18 MuiButton-textSizeSmall MuiButton-sizeSmall Label-dense-17 Mui-disabled Label-info-24 Label-disabled-19 Mui-disabled"
```

Comment:
this looks like a generated file - can we hide it from PR view like the other generated files?

Reply:
Didn't know about this feature! Neat!